setup/CLI/LiveConsoleProgress.cs

```diff
@@ -45,8 +45,8 @@ private sealed class TaskProgress : ITaskProgress
 		private readonly string description;
 		private readonly Table table;
 
-		private readonly int row;
-		private readonly int detailsRow;
+		private readonly int HeaderRow;
+		private readonly int StatusRow;
```

Comment:
@Chicken-Bones These field names don't conform to the naming of any other readonly fields in the solution. Should be lowercase.